This patches the unet3 lobby to enable the unet3 tracking protocol (the unet3 tracking server does not support
it, so it was disabled in the lobby as well). After applying this patch, you can set tracking.oldprotocol=0 if
you use the unet3+ tracking server.

diff -ur unet3_orig/gctrackingmsg.cpp unet3/gctrackingmsg.cpp
--- unet3_orig/gctrackingmsg.cpp	2007-07-02 14:11:48.000000000 +0200
+++ unet3/gctrackingmsg.cpp	2008-05-23 12:17:20.000000000 +0200
@@ -132,8 +132,8 @@
 		char * aux;
 		aux=cnf_getString("255.255.255.0","private_mask","global",global_config);
 		off+=encode_urustring(buf+off,(Byte *)aux,strlen((char*)aux),0);
-		off+=encode_urustring(buf+off,(Byte *)net->address,strlen((char*)net->address),0);
 	}
+	off+=encode_urustring(buf+off,(Byte *)net->address,strlen((char*)net->address),0);
 
 	ret=plNetSendMsg(net,buf,off,sid,0);
 	return ret;
diff -ur unet3_orig/uru.cpp unet3/uru.cpp
--- unet3_orig/uru.cpp	2007-07-02 14:11:48.000000000 +0200
+++ unet3/uru.cpp	2008-05-23 10:24:57.000000000 +0200
@@ -430,7 +430,7 @@
 	if(net->pro_vault!=0 && net->pro_vault!=1) net->pro_vault=0;
 	if(net->pro_tracking!=0 && net->pro_tracking!=1) net->pro_tracking=0;
 	if(net->pro_tracking==0) {
-		_DIE("You want to fry your tracking server?, then set tracking.oldprotocol=1\n");
+		//_DIE("You want to fry your tracking server?, then set tracking.oldprotocol=1\n");
 	}
 
 }
